### ISSUE TITLE
Release 8.8.6 and 9.1.0 unable to init a new Shield deployment

### ISSUE THREAD

**Issue by rguilme1** [shieldproject/shield-boshrelease#194]
Describe the bug
When running initial setup of a new installation of Shield, the setup fails.

To Reproduce

deploy shield/8.8.6 bosh release on new VM
login to shield web ui using admin user
Select Setup
Enter new master password twice
error message: Unable to initialize the SHIELD Core
Expected behavior
Shield initializes correctly

Screenshots

SHIELD versions (please complete the following information):

SHIELD Core: 8.8.6
SHIELD CLI: N/A
Additional context
/var/vcap/sys/logs/shield/shieldd

Fri Jan 27 19:45:07 UTC 2023 running any schema migrations
2023-01-27 19:45:08.327658361 +000 UTC /var/vcap/packages/shield/bin/shield-schema: INFO: deployed schema version 13
Fri Jan 27 19:45:08 UTC 2023 starting up shieldd (pid 7397)
2023-01-27 19:45:09.376825552 +000 UTC /var/vcap/packages/shield/bin/shieldd: ERROR: SHIELD's vault is %!s(vault.Status=1); please initialize or unlock this SHIELD core via the web UI or the CLI
2023-01-27 19:46:03.971995774 +000 UTC /var/vcap/packages/shield/bin/shieldd: ERROR: unable to read bootstrap.log: open /var/vcap/store/shield/bootstrap.log: no such file or directory
2023-01-27 19:46:16.119615991 +000 UTC /var/vcap/packages/shield/bin/shieldd: ERROR: POST /v2/init errored: 404 Not Found: no handler for route "secret/secret/archives/fixed_key". route entry not found.

**Comment by rguilme1** [comment]
I deployed the release 8.8.4 and was able to perform init.
Noted that vault did not mount kv store in 8.8.6 and 9.1.0 on a fresh deployment.

**Comment by JCL38-ORANGE** [comment]
Hello,
I am facing with the same problem. 
```
no handler for route "secret/secret/archives/525c8de5-987b-4cbe-a443-366055db1fa6". route entry not found
```
Is there a workaround for that ?
Thanks and regards,
Jean-Christophe.

**Comment by JCL38-ORANGE** [comment]
It seems that vault team have fixed it with vault 1.11.6 (https://github.com/hashicorp/vault/issues/16639).
Is it possible to have a new shield release with a bump a vault from 1.11.4 to 1.11.6 ?
Thanks and regards,
Jean-Christophe.

**Comment by wayneeseguin** [comment]
There will be a new version to bump vault this year, exactly when depends on folks availabilities and priorities.

**Comment by JCL38-ORANGE** [comment]
Hello,
Ok thanks for the answer.
Regards,
Jean-Christophe.

**Comment by JCL38-ORANGE** [comment]
Hello,
Do you plan to have this new release before the end of the year ?
Thanks and regards,
Jean-Christophe.

**Comment by norman-abramovitz** [comment]
Yes,  We are planning to have a new release before the end of year.

**Comment by JCL38-ORANGE** [comment]
Hello,
I see a new release 9.3.0-rc1 (https://github.com/shieldproject/shield-boshrelease/releases/tag/v9.3.0-rc.1).
When do you plan to cut the final release ?
Thanks and regards,
Jean-Christophe.

**Comment by JCL38-ORANGE** [comment]
Hello,
I see a new release 9.3.0-rc2 (https://github.com/shieldproject/shield-boshrelease/releases/tag/v9.3.0-rc.2).
When do you plan to cut the final release ?
If I understand well there will be a breaking change because of vault.
Thanks and regards,
Jean-Christophe.
